pkg/url/url.go

```diff
@@ -484,9 +531,23 @@ func ListIngressURL(client *kclient.Client, envSpecificInfo *envinfo.EnvSpecific
 		}
 	}
 
+	for routeName, routeURL := range routeMap {
+		_, found := localMap[routeName]
+		if found {
+			// URL is in both local env file and cluster
+			routeURL.Status.State = StateTypePushed
+			urls = append(urls, routeURL)
+		} else {
+			// URL is on the cluster but not in local env file
+			routeURL.Status.State = StateTypeLocallyDeleted
+			urls = append(urls, routeURL)
+		}
+	}
+
```

Comment:
These actions seem to be similar to the ingressMap. Can we merge the two into a single map?

Reply:
you are right. Updated.